interfaces/HO-Portal/src/app/models/person.model.ts

```diff
@@ -26,7 +26,7 @@ export class Person {
   whatsappPhoneNumber?: string;
   paymentAmountMultiplier?: number;
   preferredLanguage?: LanguageEnum;
-  customAttributes?: object;
+  paTableAttributes?: object;
```

Comment:
I'd say that "customAttributes" wasn't the best name, but "paTableAttribute" is at least worst. :P

But it depends on the porpose of this property what the "_best_" name would be.

My main comment/concern is that its name should describe the 'thing' in a slightly more generic way, because it will maybe be used in other ways as well, not only in a "table" for example..